backend/user/automation.py

```
@@ -1,145 +1,46 @@
+    board = 3  # pokerid
+    priority = "Medium"
+    summary = "Sample Summary"
+    description = "Sample Description"
+    ticket_key = "TICKET-123"
+    
+    # Generate emails user6@example.com to user20@example.com
+    emails = [f"user{i}@example.com" for i in range(6, 21)]
+    # Loop through emails and create tickets
+    pokerid=PokerBoard.objects.filter(pokerid=2).first()
+    new_tickets = []
+    for i in range (11,20):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)